bin/install-theos

```diff
@@ -0,0 +1,532 @@
+	if [[ $OLD == 1 ]]; then
+		read -p "Do you have 'https://coolstar.org/publicrepo/', 'https://repo.bingner.com/', and 'http://apt.thebigboss.org/repofiles/cydia/' installed in the sources of your jailbreak's primary package manager? [y/n]" ready
+		if [[ "${ready,,}" == "y" || "${ready,,}" == "yes" ]]; then
+			sudo apt install org.theos.dependencies -y --allow-unauthenticated --allow-downgrades && update "Dependencies have been successfully installed!" || (error "Dependency install command seems to have encountered an error. Please see the log above." && exit 3)
+		else
+			error "Please install the repos mentioned above before proceeding."
+			exit 3
+		fi
+	else
+		# Dependencies pkg is outdated, so need to cater install for Procursus vs Elucubratus
+		if [[ -f "/.procursus_strapped" ]]; then
+			read -p "Do you have 'https://apt.procursus.us' installed in the sources of your jailbreak's primary package manager? [y/n]" ready
+			if [[ "${ready,,}" == "y" || "${ready,,}" == "yes" ]]; then
+				sudo apt install ca-certificates clang coreutils curl dpkg git grep ldid make odcctools perl plutil rsync unzip xz-utils -y --allow-unauthenticated --allow-downgrades && update "Dependencies have been successfully installed!" || (error "Dependency install command seems to have encountered an error. Please see the log above." && exit 3)
+			else
+				error "Please install the repo mentioned above before proceeding."
+				exit 3
+			fi
+		else
+			read -p "Do you have 'https://apt.bingner.com' installed in the sources of your jailbreak's primary package manager? [y/n]" ready
+			if [[ "${ready,,}" == "y" || "${ready,,}" == "yes" ]]; then
+				sudo apt install ca-certificates clang coreutils curl dpkg git grep ldid make odcctools perl com.bingner.plutil rsync unzip xz -y --allow-unauthenticated --allow-downgrades && update "Dependencies have been successfully installed!" || (error "Dependency install command seems to have encountered an error. Please see the log above." && exit 3)
+			else
+				error "Please install the repo mentioned above before proceeding."
+				exit 3
+			fi
+		fi
+	fi
+
+	# Check desire for Swift support
+	update "Checking desire for Swift support..."
+	read -p "Would you like to be able to work with Swift? If so, an additional package will need to be installed. [y/n]" confirm
+	if [[ "${confirm,,}" == "y" || "${confirm,,}" == "yes" ]]; then
+		if [[ -f "/.procursus_strapped" ]]; then
+			sudo apt install swift -y --allow-unauthenticated --allow-downgrades && update "Additional Swift package installation successful!" || (error "Additional Swift package install command seems to have encountered an error. Please see the log above." && exit 3)
+		else
+			sudo apt install com.kabiroberai.swift-toolchain -y --allow-unauthenticated --allow-downgrades && update "Additional Swift package installation successful!" || (error "Additional Swift package install command seems to have encountered an error. Please see the log above." && exit 3)
```

Comment:
tbh I'm not sure how useful this is anymore since Swift 4 (which iirc is what this toolchain has) is suuuper old

Reply:
I don't think there's an alternative available for Eluc. Is something better than nothing here or is it just plain not useful at all since it's outdated?